code/datums/components/embedded.dm

```diff
@@ -239,7 +239,7 @@
 		to_chat(user, "<span class='warning'>You don't have the dexterity to do this!</span>")
 		return
 
-	if(istype(victim)) // check to see if the limb is actually exposed
+	if(ishuman(victim)) // check to see if the limb is actually exposed
```

Comment:
No it won't have any.
Why?
First of all it never worked, it would runtime one line later. So nothing lost on this front.

Also there almost the same code few lines bellow that differs only with the ishuman instead of istype check

Reply:
It wouldn't runtime because of byond, but I see your point